bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/component/Climate.java

```diff
@@ -163,8 +173,21 @@ static class ChannelConfiguration extends AbstractChannelConfiguration {
         protected @Nullable String modeStateTemplate;
         @SerializedName("mode_state_topic")
         protected @Nullable String modeStateTopic;
+        @SerializedName("modes")
```

Comment:
this line is unnecessary; you only need to add `@SerializedName` when the name in the JSON doesn't exactly match the name of the field in the Java class